model/job-management/src/main/proto/beam_job_api.proto

```diff
@@ -213,16 +213,37 @@ message JobMessagesResponse {
 // without needing to pass through STARTING.
 message JobState {
   enum Enum {
+    // The job state reported by a runner cannot be interpreted by the SDK.
     UNSPECIFIED = 0;
+
+    // The job has been paused, or has not yet started.
     STOPPED = 1;
+
+    // The job is currently running. (terminal)
     RUNNING = 2;
+
+    // The job has successfully completed. (terminal)
     DONE = 3;
+
+    // The job has failed. (terminal)
     FAILED = 4;
+
+    // The job has been explicitly cancelled. (terminal)
     CANCELLED = 5;
+
+    // The job has been updated.
     UPDATED = 6;
+
+    // The job is draining its data.
     DRAINING = 7;
+
+    // The job has completed draining its data. (terminal)
     DRAINED = 8;
+
+    // The job is starting up.
     STARTING = 9;
+
+    // The job is cancelling.
     CANCELLING = 10;
     UPDATING = 11;
```

Comment:
No comment on UPDATING?

Reply:
Ah, sorry. that was added by @lukecwik after I created this.  Will do.

Reply:
`// The job is in the progress of being updated.`

Reply:
fixed